ivy/functional/frontends/torch/tensor.py

```diff
@@ -1891,7 +1894,9 @@ def addcdiv_(self, tensor1, tensor2, *, value=1):
         ).ivy_array
         return self
 
-    @with_unsupported_dtypes({"2.2 and below": ("bfloat16", "float16")}, "torch")
+    @with_supported_dtypes(
+        {"2.2 and below": ("float32", "float64", "complex32", "complex64")}, "torch"
+    )
```

Comment:
Changing this seems to cause the torch.Tensor.cholesky test to fail with a tensorflow backend? Can we just remove this from the PR? Otherwise looks good 😁

Reply:
But the origenal version would let the test generate boolean values for cholesky, which is not right. So can I have more details on the failure caused by this? Thanks

Reply:
```
torch._C._LinAlgError: cholesky: The factorization could not be completed because the input is not positive-definite (the leading minor of order 1 is not positive-definite)
```
Looks like you need to ensure that the input matrix is positive-definite.

You should be able to reproduce the error yourself with:
```
pytest ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py::test_torch_cholesky -p no:warnings --backend tensorflow
```

Also I think `"complex128"` is a supported dtype.